global.js

```diff
@@ -161,7 +161,22 @@ if (!ts.eventsRegistered) {
 
     // Events.on(EventType.PlayerJoin, cons(e => teamKeeper(e.player, false)))
     // Events.on(EventType.PlayerLeave, cons(e => teamKeeper(e.player, true)))
-
+    
+    var lastTick = 0
+    var mean = new WindowedMean(60)
+    const timer = new Interval()
+    Events.run(Trigger.update, () -> {
+        mean.add(Time.sinceMillis(lastTick))
```

Comment:
this crashes the server
![image](https://user-images.githubusercontent.com/48705529/120091535-58a39b80-c125-11eb-8f5d-cd7ba345844f.png)

Reply:
oh rip